src/main/java/com/internet/shop/storage/Storage.java

```diff
@@ -0,0 +1,39 @@
+package com.internet.shop.storage;
+
+import com.internet.shop.model.Order;
+import com.internet.shop.model.Product;
+import com.internet.shop.model.ShoppingCart;
+import com.internet.shop.model.User;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    public static final List<User> userStorage = new ArrayList<>();
+    public static final List<Product> productStorage = new ArrayList<>();
+    public static final List<Order> orderStorage = new ArrayList<>();
+    public static final List<ShoppingCart> shoppingCartStorage = new ArrayList<>();
+    private static Long productCounter = 1L;
+    private static Long userCounter = 1L;
+    private static Long shoppingCartCounter = 1L;
+    private static Long orderCounter = 1L;
+
+    public static void addProduct(Product product) {
+        product.setId(productCounter++);
```

Comment:
```suggestion
        product.setId(++productCounter);
```
same for all methods